src/test/java/org/example/c03/controlFlow/NumberPalindromeTest.java

```diff
@@ -0,0 +1,15 @@
+package org.example.c03.controlFlow;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class NumberPalindromeTest {
+    @Test
+    public void given12321ShouldReturnTrue() {
+        int number = 12321;
+        boolean isPalindrome = true;
+        boolean result = NumberPalindrome.isPalindrome(number);
+        assertEquals(result, isPalindrome);
```

Comment:
Alternatively, remove line 11, rename result to isPalindrome  and  replace assertEquals(result, isPalindrome) with asssertTrue(isPalindrome)

Reply:
done